src/harness/harnessIO.ts

```diff
@@ -87,19 +87,20 @@ function createNodeIO(): IO {
     function listFiles(path: string, spec: RegExp, options: { recursive?: boolean; } = {}) {
         function filesInFolder(folder: string): string[] {
             let paths: string[] = [];
+            const dir = fs.opendirSync(folder);

```

Comment:
Given we're optimizing for performance, does increasing the default buffer size from 32 help? That option has been available since Node 12.

Reply:
Switched over to withFileTypes

Reply:
Thanks; does the perf improvement change at all?